crates/task-impls/src/consensus.rs

```diff
@@ -1485,6 +1496,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                     ""
                 );
 
+                #[cfg(not(feature = "proposal-task"))]
```

Comment:
Nit: Maybe it's just a temporary thing for debugging, but I don't think this and the same thing on line 1573 are needed, since this function already has this feature flag.

Reply:
It is a temporary debugging thing. This specific call not being present causes everything to break.